index.bs

```diff
@@ -8486,12 +8503,11 @@ interfaces. Specifically:
 
 </div>
 
-If the [{{Global}}] or
-[{{PrimaryGlobal}}]
-[=extended attributes=]
-is used on an [=interface=], then:
+If the [{{Global}}] or [{{PrimaryGlobal}}] [=extended attributes=] is
+used on an [=interface=], then:
 
 *   The interface must not define a [=named property setter=].
```

Comment:
Oh, so that's a bug with the spec that—I guess—has been around for a while. I don't think it has any implications here, so we might as well file a separate bug for it.